### PR TITLE
update gcp-documentation

### PR DESCRIPTION
**Pull request by ovidiuasiminei-ssc**

### CODE REVIEW

File: Architecture/Repository Structure.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ To overcome this limitation and checkout a specific tag or commit SHA, run `modu
 The tools submodule contains a `hydrate.sh` script to hydrate the configs with `kpt`.  The script must be executed when any changes to `source-base` and/or `source-customization` are made.  It is configured to run as a pre-commit hook for local validation and also in a validation pipeline during PR creation (TODO: create pipeline).
 
 At a high level, the script will:
-1. For each environment (sandbox, dev, uat, prod), check if `source-customization/` contains that sub directory.  If so:
+1. For each environment (experimentation, dev, uat, prod), check if `source-customization/` contains that sub directory.  If so:
```
Comment (alaincormier-ssc) [inline Architecture/Repository Structure.md:65]: the hydrate.sh script and repo folders are not yet updated to use the new name, maybe we should keep sandbox until they're updated?

Comment (ovidiuasiminei-ssc) [reply Architecture/Repository Structure.md:65]: Yes I don't changed. I think is best to keep until they're are updated . Thanks Alain

**Comment by alaincormier-ssc** [comment]
the `Building_Using_Automation.md` will also need an update to a sandbox package

File: Architecture/Repository Structure.md (hunk 1 above)
Comment (davelanglois-ssc) [reply Architecture/Repository Structure.md:65]: we are going forward with the modification of the documentation.  @ovidiuasiminei-ssc will create PR for gcp-tools and gcp-repo-template

Comment (ovidiuasiminei-ssc) [reply Architecture/Repository Structure.md:65]: Thanks Dave, I will create PR for gcp-tools and gcp-repo-teplate

File: Architecture/Repository Structure.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The git repos are organized in different categories:
 - Deployment repos: two types exists, one to manage Config Sync repos, the other to store infrastructure configurations.
     - `gcp-tier1-configsync` is where the initial `root-sync` object is pointing and is used to manage the root sync objects and versioning of the tier1 infrastructure in sandbox, dev, uat and prod.
     - `gcp-tier1-infra` contains the landing zone configs for dev, uat and prod.
-    - `gcp-sandbox-tier1-infra` contains the landing zone configs for sandbox.
+    - `gcp-sandbox-tier1-infra` contains the landing zone configs for experimentation.
```
Comment (davelanglois-ssc) [inline Architecture/Repository Structure.md:27]: rename repo

File: Developer/Guide.md [hunk 3]
```diff
@@ -124,7 +124,7 @@ TODO: provide an example
 
 ### Repositories naming convention
 
-The diagram below describes the naming convention for repositories and uses NRCan, as a tenant, to provide examples.
+The diagram below describes the naming convention for repositories and uses NRCan, as a client, to provide examples.
```
Comment (davelanglois-ssc) [inline Developer/Guide.md:127]: The diagram below describes the naming convention for repositories and uses na, as a client code, to provide examples

File: Onboarding/Tenant.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 
 ## Required Information
 
-1. Tenant's Name
-1. When creating the name of a tenant folder, it must adhere to a bilingual nomenclature and leverage the official abbreviation list of current Government of Canada departments, agencies, Crown Corporations and special operating agencies: [https://www.canada.ca/en/government/dept.html](https://www.canada.ca/en/government/dept.html).
+1. Client's Name
+1. When creating the name of a client folder, it must adhere to a bilingual nomenclature and leverage the official abbreviation list of current Government of Canada departments, agencies, Crown Corporations and special operating agencies: [https://www.canada.ca/en/government/dept.html](https://www.canada.ca/en/government/dept.html).
 
-   For example, when creating a tenant folder to represent Shared Services Canada, employ both the official English and French abbreviation separated by a dash (e.g. SSC-SPC).
+   For example, when creating a client folder to represent Shared Services Canada, employ both the official English and French abbreviation separated by a dash (e.g. SSC-SPC).
```
Comment (davelanglois-ssc) [inline Onboarding/Tenant.md:8]: please use lower case...e.g. .ssc-spc

File: Onboarding/Tenant.md [hunk 5]
```diff
@@ -34,21 +34,21 @@
...
     ```
     cd source-base
     ```
-1. Get the hierarchy/tenant package
-    - Sandbox
+1. Get the hierarchy/client package
+    - Experimentation
       ```
-      kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/hierarchy/tenant-sandbox@main ./landing-zone/hierarchy/Tenants/<tenant name>
+      kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/hierarchy/client-sandbox@main ./landing-zone/hierarchy/clients/<client name>
```
Comment (davelanglois-ssc) [inline Onboarding/Tenant.md:46]: replace client-sandbox

File: Onboarding/Workload.md [hunk 6]
```diff
@@ -41,14 +41,14 @@
 1. Get the workload package
     
     *Some folders may need to be created beforehand like `<data classification>`
-    - Sandbox
+    - Experimentation
       ```
-      kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/project/project-sandbox@main ./landing-zone/hierarchy/Tenants/<tenant name>/Workloads/<project-id>
+      kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/project/project-experimentation@main ./landing-zone/hierarchy/clients/<client name>/Workloads/<project-id>
```
Comment (davelanglois-ssc) [inline Onboarding/Workload.md:46]: @uday needs to update that package

Comment (davelanglois-ssc) [reply Onboarding/Workload.md:46]: rename Workloads accordingly

File: Onboarding/Workload.md [hunk 7]
```diff
@@ -41,14 +41,14 @@
...
       ```
 
     - DEV, UAT, PROD
       ```
-      kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/project/project-sandbox@main ./landing-zone/hierarchy/Tenants/<tenant name>/Workloads/<data classification>/<project-id>
+      kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/project/project-experimentation@main ./landing-zone/hierarchy/clients/<client name>/Workloads/<data classification>/<project-id>
```
Comment (davelanglois-ssc) [inline Onboarding/Workload.md:51]: rename Workloads

File: Onboarding/Workload.md [hunk 8]
```diff
@@ -32,7 +32,7 @@
 1. Locally clone the landing zone repo for this environment
 1. Create a branch from main
 
-## Add tenant's workload to the landing zone repository
+## Add client's workload to the landing zone repository
```
Comment (davelanglois-ssc) [inline Onboarding/Workload.md:35]: rename workload

**Comment by davelanglois-ssc** [comment]
@ovidiuasiminei-ssc also rename onboarding/Tenant.md to client.md and Workload.md to application.md

**Review by davelanglois-ssc — DISMISSED** [review]

**Comment by alaincormier-ssc** [comment]
hydrate.sh will now be updated to use experimentation instead of sandbox, same with the template repo.
there are reference to sandbox in these files:
- Changing.md
- Repositories.md

File: Landing Zone Operations/Building_Using_Automation.md [hunk 9]
```diff
@@ -31,7 +31,7 @@ The automated script creates a project, the FW settings, a Cloud router, a Cloud
 The script requires a `.env` file to deploy the environment.
 
 1. Start a new change for your `tier1-infra` repo, follow the "Step 1 - Setup" section of [Changing.md](./Changing.md).
-    - Sandbox
+    - Experimentation
 
         repo name = `gcp-sandbox-tier1-infra`
```
Comment (alaincormier-ssc) [inline Landing Zone Operations/Building_Using_Automation.md:36]: `gcp-sandbox-tier1-infra` should be `gcp-experimentation-tier1-infra`

also see my previous [comment](https://github.com/ssc-spc-ccoe-cei/gcp-documentation/pull/11#issuecomment-1433589907) for `Changing.md` and `Repositories.md`.
thanks

Comment (ovidiuasiminei-ssc) [reply Landing Zone Operations/Building_Using_Automation.md]: Thanks Alain, Done !

**Comment by uday-ssc** [comment]
@ovidiuasiminei-ssc  the branch name should be github specific issue , not sure if we want to strictly follow the convention tho.

**Comment by ovidiuasiminei-ssc** [comment]
I was looking what branch we have

***@***.***:~/code/gcp-documentation$ git branch -a
* az4314-gcp-doc
  main
  remotes/origin/HEAD -> origin/main
  remotes/origin/az3190-dave
  remotes/origin/az3190-update-setup-kcc
  remotes/origin/az3573-gator-cli
  remotes/origin/az3625-docs-repos
  remotes/origin/az4314-gcp-doc
  remotes/origin/doc/gcpboard-4006
  remotes/origin/main

**Comment by uday-ssc** [comment]
@ovidiuasiminei-ssc  some files are not in review , but need to change , 
1. Organizations.png (developer/img) - this image has to change
2. 
image
this has to be experimentation-networking

3.
image
this image is incorrect

**Review by uday-ssc — CHANGES_REQUESTED** [review]

File: Onboarding/Tenant.md [hunk 10]
```diff
@@ -1,11 +1,11 @@
-# Tenant Onboarding
+# Client Onboarding
```
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Onboarding/Tenant.md:1]: The file name is Client.md

https://github.com/ssc-spc-ccoe-cei/gcp-documentation/pull/11/files/3d57a6a9e409a151817aa58ba368dbfebfa809db

File: Onboarding/Admin.md [hunk 11]
```diff
@@ -2,7 +2,7 @@
 
 ## Pre-requisite
 
-1. locally clone the sandbox landing zone repo
+1. locally clone the experimentation landing zone repo
```
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Onboarding/Admin.md:5]: Fixed

File: Onboarding/Admin.md [hunk 12]
```diff
@@ -2,7 +2,7 @@
...
 1. create a branch of main
```
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Onboarding/Admin.md:6]: Fixed

File: Onboarding/Admin.md [hunk 13]
```diff
@@ -2,7 +2,7 @@
...
 
 ## Add admin folder(s) to the landing zone repository
```
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Onboarding/Admin.md:8]: Fixed

File: Developer/Guide.md [hunk 14]
```diff
@@ -124,7 +124,7 @@ TODO: provide an example
 
 ### Repositories naming convention
 
-The diagram below describes the naming convention for repositories and uses NRCan, as a tenant, to provide examples.
+The diagram below describes the naming convention for repositories and uses na, as a client, to provide examples.
```
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Developer/Guide.md:127]: fixed

File: Onboarding/Application.md [hunk 15]
```diff
@@ -3,10 +3,10 @@
 ## Required Information
 ### Common
 
-1. Naming convention for project-id : `<tenant-code><environment-code><region-code><data-classification>`-`<project-owner>`-`<user defined string>`
+1. Naming convention for project-id : `<client-code><environment-code><region-code><data-classification>`-`<project-owner>`-`<user defined string>`
```
[the remark this answers is not in the document]
Comment (ovidiuasiminei-ssc) [reply Onboarding/Application.md:6]: Fixed

**Comment by davelanglois-ssc** [comment]
> ame should be github specific iss

@ovidiuasiminei-ssc @uday-ssc Well, I believe this is not the case for our SSC repos. Right now, we don't have any issues open on our repos so we can avoid that burden of replicating the azure devops storys into github and just use the azdo story number for the branches. The situation is different on the pubsec repos as this is a repo with an existing long list of issues and, it also has much more visibility than ours meaning that we will put in the additional effort of generating a github issue for the azdo stories. Starting next sprint, @cheffj  will be responsible of tracking/synching azdo/github for the pubsec repo

**Review by uday-ssc — APPROVED** [review]

**Review by davelanglois-ssc — APPROVED** [review]

**Pull request merged**